### DISCUSSION TITLE
I tried to Open modal for second step , using with  MulitiRoute joyride it's not working

### DISCUSSION THREAD

**Discussion by Jiilakara** [gilbarbara/react-joyride#1058] [General]
Hi @gilbarbara 

Please find the below are my sample steps  
steps: [
          {
            target: "#example1",
            content: "Welcome to the Application.",
            disableBeacon: true,
          },
          {
            target: "#example2",
            content: "This is the Add modal. It will open automatically when this step is reached.",
            disableBeacon: true,
            data: { next: "/next" },
          },
          {
            target: "#example3",
            content: "This is the upload page. Please upload your document here.",
            data: { previous: "/previous", next: "/next" },
            disableBeacon: true,
          },
          {
            target: "#example3",
            content: "This is the example page",
            data: { previous: "/next", next: "/previous" },
            disableBeacon: true,
          },
          {
            target: "#example4",
            content: "This is example page",
            disableBeacon: true,
          },
        ],

In second step  calling modal  but it is not opening . kindly help

**Comment by gilbarbara** [comment]
Hey @Jiilakara 

You have already created a discussion about this problem, so please read my reply and answer in the same discussion.
